engine/src/main/java/org/camunda/bpm/engine/management/JobDefinition.java

```diff
@@ -97,4 +97,10 @@
    */
   String getTenantId();
 
+  /**
+   * The id of the deployment this job is related to. In a deployment-aware setup, this
```

Comment:
Sounds like the documentation of `Job#getDeploymentId` and not of `JobDefinition`.

Reply:
I will adjust it a bit 👍